aptos-move/block-executor/src/executor.rs

```diff
@@ -1264,8 +1422,13 @@ where
                 PanicOr::Or(IntentionalFallbackToSequential::ModulePathReadWrite) => {
                     panic!("ModulePathReadWrite shouldn't happen in sequential execution")
```

Comment:
Should this be.... `error!` Or at least `unreachable!` (which is a panic but it reads less scary)?

Reply:
unreachable is perfect here.